src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -102,10 +97,10 @@ public void setMax(int max) {
             TokenTypes.LITERAL_IF,
             TokenTypes.LITERAL_ELSE,
             TokenTypes.LITERAL_SWITCH,
-            TokenTypes.LITERAL_CASE,
-            TokenTypes.LITERAL_TRY,
-            TokenTypes.LITERAL_CATCH,
```

Comment:
@romani I don't know NPath but should try/catch be removed?
I assume `case` was converted to `case_group` as I see it counting cases below.